src/test/java/uk/ac/ebi/eva/pipeline/jobs/steps/VepInputGeneratorStepTest.java

```diff
@@ -73,15 +80,24 @@
 
     @Test
     public void shouldGenerateVepInput() throws Exception {
+        checkVepInputWasGenerated(STUDY_ID);
```

Comment:
I prefer to have the assert in the test method.. but this is just my personal preference. I found it easier to understand even if I have to add a bit of code duplication.